components/Comment/CommentFeed.tsx

```diff
@@ -17,33 +17,55 @@ import CommentFilters from "./CommentFilters";
 import { css, StyleSheet } from "aphrodite";
 import { filterOpts, sortOpts } from "./lib/options";
 import CommentSort from "./CommentSort";
-
+import CommentPlaceholder from "./CommentPlaceholder";
+import config from "./lib/config";
+import colors from "./lib/colors";
 
 type Args = {
-  unifiedDocumentId: ID;
+  document: TopLevelDocument;
+  WrapperEl: any;
 };
 
-const CommentFeed = ({ unifiedDocumentId }: Args) => {
+const CommentFeed = ({ document, WrapperEl = React.Fragment }: Args) => {
   const [comments, setComments] = useState<CommentType[]>([]);
-  const [isFetching, setIsFetching] = useState<boolean>(true);
+  const [isFetching, setIsFetching] = useState<boolean>(false);
```

Comment:
We have 3 fetch render states here. Is it possible to boil it down to just 1? Feels harder to maintain with 3 states that might be able to be set with 1.